### YamlMime:ManagedReference
items:
- uid: System.Runtime.DurableInstancing.InstanceCompleteException
  id: InstanceCompleteException
  children:
  - System.Runtime.DurableInstancing.InstanceCompleteException.#ctor
  - System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.String)
  - System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.String,System.Exception)
  - System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Xml.Linq.XName,System.Guid)
  - System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Xml.Linq.XName,System.Guid,System.Exception)
  - System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Xml.Linq.XName,System.Guid,System.String,System.Exception)
  langs:
  - csharp
  name: InstanceCompleteException
  nameWithType: InstanceCompleteException
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException
  type: Class
  summary: "Olyan adatmegőrzési szolgáltatót ezt a kivételt jelez, amikor várhatóan található egy példány inicializált állapotban, de talál, a példány a befejezett állapotban van."
  syntax:
    content: 'public class InstanceCompleteException : System.Runtime.DurableInstancing.InstancePersistenceCommandException'
  inheritance:
  - System.Exception
  - System.Runtime.DurableInstancing.InstancePersistenceException
  - System.Runtime.DurableInstancing.InstancePersistenceCommandException
  implements: []
  inheritedMembers:
  - System.Runtime.DurableInstancing.InstancePersistenceCommandException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Runtime.DurableInstancing.InstancePersistenceCommandException.InstanceId
  - System.Runtime.DurableInstancing.InstancePersistenceException.CommandName
  platform:
  - net462
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor
  id: '#ctor'
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  langs:
  - csharp
  name: InstanceCompleteException()
  nameWithType: InstanceCompleteException.InstanceCompleteException()
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException()
  type: Constructor
  assemblies:
  - System.Runtime.DurableInstancing
  namespace: System.Runtime.DurableInstancing
  summary: "Egy példányát inicializálja a <xref href=&quot;System.Runtime.DurableInstancing.InstanceCompleteException&quot;> </xref> osztály."
  syntax:
    content: public InstanceCompleteException ();
    parameters: []
  overload: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  langs:
  - csharp
  name: InstanceCompleteException(String)
  nameWithType: InstanceCompleteException.InstanceCompleteException(String)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(String)
  type: Constructor
  assemblies:
  - System.Runtime.DurableInstancing
  namespace: System.Runtime.DurableInstancing
  summary: "Egy példányát inicializálja a <xref href=&quot;System.Runtime.DurableInstancing.InstanceCompleteException&quot;> </xref> osztály használata a hibaüzenetet."
  syntax:
    content: public InstanceCompleteException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel okát."
  overload: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  langs:
  - csharp
  name: InstanceCompleteException(SerializationInfo,StreamingContext)
  nameWithType: InstanceCompleteException.InstanceCompleteException(SerializationInfo,StreamingContext)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Runtime.DurableInstancing
  namespace: System.Runtime.DurableInstancing
  summary: "Egy példányát inicializálja a <xref href=&quot;System.Runtime.DurableInstancing.InstanceCompleteException&quot;> </xref> a szerializált adatok osztályt."
  syntax:
    content: protected InstanceCompleteException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A jelenlegi kivétel információt."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A cél- és környezeti információt."
  overload: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  langs:
  - csharp
  name: InstanceCompleteException(String,Exception)
  nameWithType: InstanceCompleteException.InstanceCompleteException(String,Exception)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(String,Exception)
  type: Constructor
  assemblies:
  - System.Runtime.DurableInstancing
  namespace: System.Runtime.DurableInstancing
  summary: "Egy példányát inicializálja a <xref href=&quot;System.Runtime.DurableInstancing.InstanceCompleteException&quot;> </xref> osztály a hibaüzenet és a belső kivételben találhatók információk."
  syntax:
    content: public InstanceCompleteException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel okát."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy az aktuális kivételt okoz."
  overload: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Xml.Linq.XName,System.Guid)
  id: '#ctor(System.Xml.Linq.XName,System.Guid)'
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  langs:
  - csharp
  name: InstanceCompleteException(XName,Guid)
  nameWithType: InstanceCompleteException.InstanceCompleteException(XName,Guid)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(XName,Guid)
  type: Constructor
  assemblies:
  - System.Runtime.DurableInstancing
  namespace: System.Runtime.DurableInstancing
  summary: "Egy példányát inicializálja a <xref href=&quot;System.Runtime.DurableInstancing.InstanceCompleteException&quot;> </xref> osztály XName (a névtér és a parancs nevét kombinációja) és a célpéldányon, amelyre vonatkozóan a parancs végrehajtása történt Azonosítóját."
  syntax:
    content: public InstanceCompleteException (System.Xml.Linq.XName commandName, Guid instanceId);
    parameters:
    - id: commandName
      type: System.Xml.Linq.XName
      description: "A XName (kombinációja névtere és neve), a parancsot."
    - id: instanceId
      type: System.Guid
      description: "A célpéldányon, amelyre vonatkozóan a parancs végrehajtása történt azonosítója."
  overload: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Xml.Linq.XName,System.Guid,System.Exception)
  id: '#ctor(System.Xml.Linq.XName,System.Guid,System.Exception)'
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  langs:
  - csharp
  name: InstanceCompleteException(XName,Guid,Exception)
  nameWithType: InstanceCompleteException.InstanceCompleteException(XName,Guid,Exception)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(XName,Guid,Exception)
  type: Constructor
  assemblies:
  - System.Runtime.DurableInstancing
  namespace: System.Runtime.DurableInstancing
  summary: "Egy példányát inicializálja a <xref href=&quot;System.Runtime.DurableInstancing.InstanceCompleteException&quot;> </xref> osztály XName (kombinációja névtere és neve), a parancs a célpéldányon, amelyre vonatkozóan a parancs végrehajtása történt, és a belső kivétel paraméterként azonosító használatával."
  syntax:
    content: public InstanceCompleteException (System.Xml.Linq.XName commandName, Guid instanceId, Exception innerException);
    parameters:
    - id: commandName
      type: System.Xml.Linq.XName
      description: "A XName (kombinációja névtere és neve), a parancsot."
    - id: instanceId
      type: System.Guid
      description: "A célpéldányon, amelyre vonatkozóan a parancs végrehajtása történt azonosítója."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy az aktuális kivételt okozott."
  overload: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Xml.Linq.XName,System.Guid,System.String,System.Exception)
  id: '#ctor(System.Xml.Linq.XName,System.Guid,System.String,System.Exception)'
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  langs:
  - csharp
  name: InstanceCompleteException(XName,Guid,String,Exception)
  nameWithType: InstanceCompleteException.InstanceCompleteException(XName,Guid,String,Exception)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(XName,Guid,String,Exception)
  type: Constructor
  assemblies:
  - System.Runtime.DurableInstancing
  namespace: System.Runtime.DurableInstancing
  summary: "Egy példányát inicializálja a <xref href=&quot;System.Runtime.DurableInstancing.InstanceCompleteException&quot;> </xref> osztály XName (kombinációja névtere és neve), a parancs a célpéldány elleni Azonosítójának használatával, amely a parancs végrehajtása történt, hibaüzenet, amely ismerteti a kivétel okát és a kivétellel, hogy az aktuális kivételt okozott."
  syntax:
    content: public InstanceCompleteException (System.Xml.Linq.XName commandName, Guid instanceId, string message, Exception innerException);
    parameters:
    - id: commandName
      type: System.Xml.Linq.XName
      description: "A XName (kombinációja névtere és neve), a parancsot."
    - id: instanceId
      type: System.Guid
      description: "A célpéldányon, amelyre vonatkozóan a parancs végrehajtása történt azonosítója."
    - id: message
      type: System.String
      description: "A kivétel okát."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy az aktuális kivételt okozott."
  overload: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Runtime.DurableInstancing.InstancePersistenceCommandException
  isExternal: false
  name: System.Runtime.DurableInstancing.InstancePersistenceCommandException
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  isExternal: false
  name: InstanceCompleteException()
  nameWithType: InstanceCompleteException.InstanceCompleteException()
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException()
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.String)
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  isExternal: false
  name: InstanceCompleteException(String)
  nameWithType: InstanceCompleteException.InstanceCompleteException(String)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  isExternal: false
  name: InstanceCompleteException(SerializationInfo,StreamingContext)
  nameWithType: InstanceCompleteException.InstanceCompleteException(SerializationInfo,StreamingContext)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.String,System.Exception)
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  isExternal: false
  name: InstanceCompleteException(String,Exception)
  nameWithType: InstanceCompleteException.InstanceCompleteException(String,Exception)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Xml.Linq.XName,System.Guid)
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  isExternal: false
  name: InstanceCompleteException(XName,Guid)
  nameWithType: InstanceCompleteException.InstanceCompleteException(XName,Guid)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(XName,Guid)
- uid: System.Xml.Linq.XName
  parent: System.Xml.Linq
  isExternal: true
  name: XName
  nameWithType: XName
  fullName: System.Xml.Linq.XName
- uid: System.Guid
  parent: System
  isExternal: true
  name: Guid
  nameWithType: Guid
  fullName: System.Guid
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Xml.Linq.XName,System.Guid,System.Exception)
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  isExternal: false
  name: InstanceCompleteException(XName,Guid,Exception)
  nameWithType: InstanceCompleteException.InstanceCompleteException(XName,Guid,Exception)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(XName,Guid,Exception)
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor(System.Xml.Linq.XName,System.Guid,System.String,System.Exception)
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  isExternal: false
  name: InstanceCompleteException(XName,Guid,String,Exception)
  nameWithType: InstanceCompleteException.InstanceCompleteException(XName,Guid,String,Exception)
  fullName: System.Runtime.DurableInstancing.InstanceCompleteException.InstanceCompleteException(XName,Guid,String,Exception)
- uid: System.Runtime.DurableInstancing.InstanceCompleteException.#ctor*
  parent: System.Runtime.DurableInstancing.InstanceCompleteException
  isExternal: false
  name: InstanceCompleteException
  nameWithType: InstanceCompleteException.InstanceCompleteException
